### PR TITLE
feat(asana): add 'addTask' method to Sections interface as per asana@0.17.3

### PR DESCRIPTION
**Pull request by filipposarzana**
- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [ ] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

---

- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://github.com/Asana/node-asana/blob/v0.17.3/lib/resources/gen/sections.js#L142
- [x] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.
- [ ] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`. If for reason the any rule need to be disabled, disable it for that line using `// tslint:disable-next-line [ruleName]` and not for whole package so that the need for disabling can be reviewed.

By looking into `asana` code, types definition could seems wrong (variable name mismatch: they call `task` what actually should be `section`). This is up-to-date documentation: https://developers.asana.com/docs/#add-task-to-section

### CODE REVIEW

File: types/asana/index.d.ts [hunk 1]
```diff
@@ -1970,6 +1977,22 @@ declare namespace asana {
...
+         *   * @param {String} data.insert_after An existing task within this section after which the added task should be inserted. Cannot be provided together with insert_before.
+         *   * @param {String} data.insert_before An existing task within this section before which the added task should be inserted. Cannot be provided together with insert_after.
+         *   * @param {Object} [dispatchOptions] Options, if any, to pass the dispatcher for the request
+         *   * @return {Promise} The response from the API
+         * @param project
+         * @param data
+         * @param dispatchOptions?
+         * @return
+         */
+        addTask(section: string | number, data: Sections.SectionsParams, dispatchOptions?: any): Promise<{}>;
```
Comment (rbuckton) [inline types/asana/index.d.ts]: Is there a better type for the promise than `{}`?

Comment (filipposarzana) [reply types/asana/index.d.ts]: @rbuckton Actually this is what Asana `addTask` resolves to 🤷‍♂

Comment (rbuckton) [reply types/asana/index.d.ts]: The `@param` tags are duplicated and the JSDoc formatting seems to be off, once that's addressed I can merge

**Review by rbuckton — CHANGES_REQUESTED** [review]
Please fix the doc comments on `addTask`

File: types/asana/index.d.ts (hunk 1 above)
Comment (filipposarzana) [reply types/asana/index.d.ts]: Thanks @rbuckton and sorry for indentation, did not even notice that 😥

**Comment by filipposarzana** [comment]
Hi @rbuckton have got any chance of reviewing this? 🙏 Thanks!

**Comment by filipposarzana** [comment]
@rbuckton Sorry for pushing this

**Pull request merged**